packages/crayons-core/package.json

```diff
@@ -69,6 +69,7 @@
     "@semantic-release/git": "^9.0.0",
     "@stencil/postcss": "^2.1.0",
     "@stencil/sass": "^1.4.1",
+    "@types/google-libphonenumber": "^7.4.23",
```

Comment:
We done seem to be using the types from this package. Can you kindly confirm if this is necessary.

Reply:
The build is failing when deleting this type.